src/k2/cmd/tpcc/transactions.h

```diff
@@ -241,8 +241,72 @@ class PaymentT : public TPCCTxn
         return writeRow<History>(history, _txn).discard_result();
     }
 
+    future<> getCIdByLastNameViaIndex() {
+        uint32_t cid_type = _random.UniformRandom(1, 100);
+        if (cid_type > 60) {
```

Comment:
It should be 60% chance to select by last name, so this should be <= 60. This bug existed in the previous code but we might as well fix it now.

Reply:
fixed and changed it to <=60